actions/authActions.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  setAuthToken: function(token) {
+    return { type: 'SET_AUTHTOKEN', token };
```

Comment:
It never has a hyphen anywhere else, it's normally AuthToken, but the convention here is to use only capital letters and underscores, hence "AUTHTOKEN".